atom/browser/extensions/atom_extensions_browser_client.cc

```diff
@@ -0,0 +1,330 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "atom/browser/extensions/atom_extensions_browser_client.h"
+
+#include <utility>
+
+#include "atom/browser/extensions/api/runtime/atom_runtime_api_delegate.h"
+#include "atom/browser/extensions/atom_extension_host_delegate.h"
+#include "atom/browser/extensions/atom_extension_system_factory.h"
+#include "atom/browser/extensions/atom_extension_web_contents_observer.h"
+#include "base/bind.h"
+#include "base/memory/ptr_util.h"
+#include "base/task/post_task.h"
+#include "build/build_config.h"
+#include "components/version_info/version_info.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/browser_task_traits.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/browser/render_frame_host.h"
+#include "content/public/browser/resource_request_info.h"
+#include "content/public/common/user_agent.h"
+#include "extensions/browser/api/extensions_api_client.h"
+#include "extensions/browser/component_extension_resource_manager.h"
+// #include "extensions/browser/core_extensions_browser_api_provider.h"
+#include "extensions/browser/event_router.h"
+#include "extensions/browser/mojo/interface_registration.h"
+#include "extensions/browser/null_app_sorting.h"
+#include "extensions/browser/updater/null_extension_cache.h"
+#include "extensions/browser/url_request_util.h"
+#include "extensions/common/features/feature_channel.h"
+// #include "atom/browser/extensions/atom_extensions_api_client.h"
+// #include "atom/browser/extensions/atom_extensions_browser_api_provider.h"
```

Comment:
Are these needed? If not, delete

Reply:
It's needed if the `AtomExtensionsBrowserAPIProvider` instance is required, on line 54.